lib/ansible/modules/packaging/os/flatpak.py

```diff
@@ -0,0 +1,256 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+# Copyright (c) 2017 Ansible Project
+# GNU General Public License v3.0+ (see COPYING or
+# https://www.gnu.org/licenses/gpl-3.0.txt)
+
+ANSIBLE_METADATA = {'metadata_version': '1.0',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+DOCUMENTATION = '''
+---
+module: flatpak
+version_added: "2.4"
+requirements:
+    - flatpak
+author:
+    - "John Kwiatkoski (@jaykayy)"
+short_description: "Install and remove flatpaks."
+description:
+    - "The flatpak module allows users to manage installation and removal of flatpaks."
+options:
+  name:
+    description:
+      - "When state is set to `present`, `name` is best used as `http(s)` url format.
```

Comment:
You don't have to quote it even if the text spans multiple lines. Also, it should use the standard formatting for names and values of options and modules as described [here](http://docs.ansible.com/ansible/latest/dev_guide/developing_modules_documenting.html#formatting-options).